client/rest/build.gradle

```diff
@@ -102,8 +102,13 @@ if (isIdea == false && isEclipse == false) {
   // removes the -nodeps jar
   configurations.runtime.artifacts.removeAll { it.archiveTask.is jar }
 
-  // removes the runtime configuration inheritance from compile
+  // runtimeElements does not exist in 3.3
+  configurations.maybeCreate('runtimeElements')
```

Comment:
I could do that too. This also works w/o any conditional logic tho for all the 3.x and 4.0. Those comments are more notes mentioning why we are removing each one. Im fine either way with it.